### DISCUSSION TITLE
Issues with Import_ and Upload Modules.

### DISCUSSION THREAD

**Discussion by alanmlawrence01** [databricks/databricks-sdk-py#551] [Q&A]
I've been using both the upload and import_ functions to upload entire directory contents into Databricks. When I run the following code I get this error: cannot import name 'ImportFormat' from 'databricks.sdk.service.workspace' (/databricks/python/lib/python3.10/site-packages/databricks/sdk/service/workspace.py). I haven't been able to get the import_ to work either does anyone have an implementation example?

            import io
            import time

            from databricks.sdk import WorkspaceClient
            from databricks.sdk.service.workspace import ImportFormat

            w = WorkspaceClient()

            py_file = f'/Users/{w.current_user.me().user_name}/file-{time.time_ns()}.py'

            w.workspace.upload(py_file, io.BytesIO(b'print(1)'), format=ImportFormat.AUTO)
            with w.workspace.download(py_file) as f:
                content = f.read()
                assert content == b'print(1)'

            w.workspace.delete(py_file)

**Comment by lkhaskin** [comment]
Experience the same problem
